proposals/csharp-8.0/default-interface-methods.md

```diff
@@ -109,11 +109,11 @@ interface IA
 }
 interface IB : IA
 {
-    override void IA.M() { WriteLine("IB.M"); } // explicitly named
+    void IA.M() { WriteLine("IB.M"); } // explicitly named
 }
 interface IC : IA
 {
-    override void M() { WriteLine("IC.M"); } // implicitly named
+    void M() { WriteLine("IC.M"); } // implicitly named
```

Comment:
>void M() { WriteLine("IC.M"); } // implicitly named [](http://example.com/codeflow?start=4&length=51)

This declares a new method in IC that has no relation to IA.M and doesn't implement it. I think the original example meant to express syntax for implicit implementation. There is no syntax that can be used for an implicit implementation in a derived interface. It looks like we can simply remove the IC interface from this example.